libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -819,6 +830,10 @@ void IRGenerator::generateConstructors(ContractDefinition const& _contract)
 				for (ASTPointer<VariableDeclaration> const& varDecl: contract->constructor()->parameters())
 					params += m_context.addLocalVariable(*varDecl).stackSlots();
 
+			if (contract->constructor())
+				t("astIDComment", "/// @ast-id " + to_string(contract->constructor()->id()));
+			else
+				t("astIDComment", "");
```

Comment:
Would it make sense to add the "\n" for the `astIDComment` so that we only have a new line when there is content?

Reply:
Yeah, thought about that, too. Will change.